src/NServiceBus.AwsLambda.SQS/TransportWrapper/ServerlessTransportInfrastructure.cs

```diff
@@ -1,56 +1,27 @@
 ﻿namespace NServiceBus.AwsLambda.SQS.TransportWrapper
 {
-    using System;
-    using System.Collections.Generic;
-    using Routing;
-    using Settings;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
     using Transport;
 
-    class ServerlessTransportInfrastructure<TBaseInfrastructure> : TransportInfrastructure
+    class ServerlessTransportInfrastructure<TBaseInfrastructure> : TransportInfrastructure where TBaseInfrastructure : TransportDefinition
```

Comment:
is the generic needed? `TBaseInfrastructure` shouldn't be of type `TransportDefinition` but I think this doesn't seem to be needed at all.

Reply:
Agreed the generic is not required